packages/framework/src/Resources/config/directories.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
+    shopsys.framework.default_public_directories:
+    - '%shopsys.web_dir%/%shopsys.content_dir_name%/feeds'
+    - '%shopsys.web_dir%/%shopsys.content_dir_name%/sitemaps'
+    - '%shopsys.web_dir%/%shopsys.content_dir_name%/wysiwyg'
+    shopsys.framework.default_internal_directories:
+    - '%kernel.project_dir%/build/stats'
+    - '%kernel.project_dir%/docs/generated'
+    - '%kernel.project_dir%/var/cache'
+    - '%kernel.project_dir%/var/lock'
+    - '%kernel.project_dir%/var/logs'
+    - '%kernel.project_dir%/var/errorPages'
+    - '%kernel.project_dir%/web/assets/admin/styles'
```

Comment:
It would make sense to me, that these would use `shopsys.web_dir` param instead

Reply:
these? I see one. :D